src/main/java/com/rapid7/presto/armor/ArmorMetadata.java

```diff
@@ -95,9 +98,11 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
         ArmorTableHandle handle = (ArmorTableHandle) table;
         try {
         	String org = handle.getSchema();
-	        List<ColumnMetadata> columns = armorClient.getColumnIds(org, handle.getTableName()).stream()
-	            .map(column -> toColumnMetaData(column))
-	            .collect(toImmutableList());
+        	List<ColumnId> columnIds = armorClient.getColumnIds(org, handle.getTableName(), Interval.SINGLE, Instant.now());
+            columnIds.add(new ColumnId("__timestamp", DataType.STRING.name()));
```

Comment:
This part may be confusing...since timestamp i not a real persisted value. I think we should rename this to be like the start interval timestamp, something to that effect. I was thinking this would be simliar to druid's segment start value.